src/t2iapi/device/types.proto

```diff
@@ -79,3 +79,12 @@ enum CalibrationState{
     CALIBRATION_STATE_CALIBRATED = 3;
     CALIBRATION_STATE_OTHER = 4;
 }
+
+/*
+Represents a container for MDIB version attributes.
+ */
+message MdibVersion {
+    string sequence_id = 1;     // SequenceId when finishing the rpc
+    uint64 instance_id = 2;     // InstanceId when finishing the rpc
```

Comment:
Is optional in BICEPS, should also be optional here, i.e. be [UInt64Value](https://protobuf.dev/reference/protobuf/google.protobuf/#uint64-value)

Reply:
done

Reply:
done